pkgs/repo_manage/README.md

```diff
@@ -0,0 +1,20 @@
+Various and miscellaneous commands to query dart-lang github repos.
+
+## Usage
+
+```
+Usage: report <command> [arguments]
+
+Global options:
+-h, --help    Print this usage information.
+
+Available commands:
+  branches          Show the default branch names of Dart and Flutter repos.
+  labels            Report on the various labels in use by dart-lang repos.
+  labels-update     Audit and update the labels used by dart-lang repos.
+  links             List useful GitHub query urls.
```

Comment:
as an aside, I may remove this command later; it's really not as useful as the other commands